manifests/init.pp

```diff
@@ -50,6 +50,11 @@
   # Users
   Optional[Hash] $users = undef,
 
+  # Ssl
+  Enum['enable','disable'] $ssl  = 'disable',
```

Comment:
Datatypes \o/

Reply:
What do you mean when you are talking about Datatypes ?

Reply:
@bastelfreak is always happy to see PRs using datatypes.